tests/test_new_cfp.py

```diff
@@ -32,7 +35,6 @@ def test_if_cfp_pages_are_login_required(client, url):
     assert redirects_to(response, "/accounts/login/")
 
 
-@mark.django_db
```

Comment:
:+1:  
what we could also do is make `user_client` fixture explicitly depend on `db` fixture which should do the same in a slightly more explicit way.

Reply:
Would that mean that all `django_db` tests would have to accept `user_client` as an argument?

Reply:
other way around – user_client would take db as argument, which it needs anyway because it creates users.